__tests__/app.test.js

```diff
@@ -45,4 +45,46 @@ beforeEach(() => {
       })
     })
 
+})
+
+//GET /api/articles/:article_id
+describe ('GET /api/articles/:article_id',() => {
+    test ('200:should return a single article by article_id',() => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then (({body}) => {
+        const {article} = body;
+
+        expect(article).toHaveLength(1);
+        expect(article[0]).toHaveProperty("author", expect.any(String));
+        expect(article[0]).toHaveProperty("title", expect.any(String));
+        expect(article[0]).toHaveProperty("article_id", expect.any(Number));
+        expect(article[0]).toHaveProperty("body", expect.any(String));
+        expect(article[0]).toHaveProperty("topic", expect.any(String));  
+        expect(article[0]).toHaveProperty("created_at");    
+        expect(article[0]).toHaveProperty("votes", expect.any(Number));   
+        expect(article[0]).toHaveProperty("article_img_url", expect.any(String));   
+    })
```

Comment:
This is great over all, however article should not be an array, you should be able to access the properties directly from article instead of having to first go to the 0 index to access any of them.